go.mod

```diff
@@ -48,6 +48,13 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
```

Comment:
why did you add this set as a separate require block?
`github.com/minio/minio-go/v7 v7.0.39` should be in the main (above) require block. All others will be automatically added in the second (// indirect) block.

Reply:
I did not make any manual changes, something went wrong during "go mod tidy" - I've removed downloaded packages and re-ran.